Market sim overnight run core error

More on https://jenkins.vega.rocks/blue/organizations/jenkins/common%2Fvega-market-sim-reinforcement/detail/vega-market-sim-reinforcement/516/pipeline 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x1864f41]

goroutine 20107 [running]:
code.vegaprotocol.io/vega/core/processor.(*App).prepareProposal(0xc000b85c00, 0x411f8c?, {0xc002bc8500, 0x12b, 0xa60000c00513ce00?}, {0xc004dea000, 0x12b, 0x9b?})
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/core/processor/abci.go:1135 +0x3d01
code.vegaprotocol.io/vega/core/blockchain/abci.(*App).PrepareProposal(0xc00067f080, {0xa8?, 0x3bbb780?}, 0xc003fd80b0)
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/core/blockchain/abci/abci.go:74 +0x355
code.vegaprotocol.io/vega/cmd/vega/commands/node.(*appW).PrepareProposal(0xc0070c1d50?, {0x473c960?, 0x6c47f60?}, 0xc001c05d40?)
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/cmd/vega/commands/node/app_wrapper.go:83 +0x25
code.vegaprotocol.io/vega/core/blockchain/nullchain.(*NullBlockchain).processBlock(0xc000a26000)
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/core/blockchain/nullchain/nullchain.go:193 +0x182
code.vegaprotocol.io/vega/core/blockchain/nullchain.(*NullBlockchain).ForwardTime(0xc000a26000, 0x3b9aca00)
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/core/blockchain/nullchain/nullchain.go:289 +0x1ce
code.vegaprotocol.io/vega/core/blockchain/nullchain.(*NullBlockchain).handleForwardTime.func1()
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/core/blockchain/nullchain/server.go:132 +0x25
created by code.vegaprotocol.io/vega/core/blockchain/nullchain.(*NullBlockchain).handleForwardTime in goroutine 20151
	/jenkins/workspace/common/vega-market-sim-reinforcement/extern/vega/core/blockchain/nullchain/server.go:131 +0x22f

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -41,6 +41,7 @@
 - [11319](https://github.com/vegaprotocol/vega/issues/11319) - Do not leak Ethereum client secrets in the logs.
 - [11336](https://github.com/vegaprotocol/vega/issues/11336) - Add support for decay factor in governance recurring transfers and report the proposal amount rather than 0 when the proposal gets enacted.
 - [11368](https://github.com/vegaprotocol/vega/issues/11368) - Add support for update vesting stats in REST API and fix summing the quantum balance for vesting stats.
+- [11380](https://github.com/vegaprotocol/vega/issues/11380) - Handle broken stop orders in prepare proposal. 
 - [11136](https://github.com/vegaprotocol/vega/issues/11136) - Fix premature invocation of post commit hooks in case of fee stats event.
 
 ## 0.76.1

--- a/core/processor/abci.go
+++ b/core/processor/abci.go
@@ -1127,12 +1127,12 @@ func (app *App) prepareProposal(height uint64, txs []abci.Tx, rawTxs [][]byte) [
 			}
 			if !someMarketRequiresDelay {
 				for _, s := range batch.StopOrdersSubmission {
-					if s.FallsBelow != nil && app.txCache.IsDelayRequired(s.FallsBelow.OrderSubmission.MarketId) {
+					if s.FallsBelow != nil && s.FallsBelow.OrderSubmission != nil && app.txCache.IsDelayRequired(s.FallsBelow.OrderSubmission.MarketId) {
 						someMarketRequiresDelay = true
 						break
 					}
 					if !someMarketRequiresDelay {
-						if s.RisesAbove != nil && app.txCache.IsDelayRequired(s.RisesAbove.OrderSubmission.MarketId) {
+						if s.RisesAbove != nil && s.RisesAbove.OrderSubmission != nil && app.txCache.IsDelayRequired(s.RisesAbove.OrderSubmission.MarketId) {
 							someMarketRequiresDelay = true
 							break
 						}

